pageserver/src/import_datadir.rs

```diff
@@ -620,6 +628,16 @@ async fn import_file(
         // TODO Backups exported from neon won't have pg_tblspc, but we will need
         // this to import arbitrary postgres databases.
         bail!("Importing pg_tblspc is not implemented");
+    } else if import_aux_files
+        && (file_path.starts_with("pg_logical/")
+            || file_path.starts_with("pg_replslot/")
+            || file_path.starts_with("pg_stat/"))
+    {
+        let bytes = read_all_bytes(reader).await?;
+        modification
+            .put_file(file_path.to_str().unwrap(), &bytes, ctx)
+            .await?;
+        debug!("imported aux file \"{}\"", file_path.display());
```

Comment:
A size limit would be good here too.

Reply:
So with this, we'll also start importing pg_logical/ and pg_replslot/ aux files. I guess that's OK? We don't expose this functionality to users currently, so I guess it doesn't matter either way in practice.

Reply:
> Actually import from pgduir happens only once. It seems not to be a big problem if we once import large file in our storage, does it?

well I'm not sure, but let's try to avoid creating special cases like this. Better safe than sorry.